uwsgi.h

```diff
@@ -4894,7 +4894,7 @@ void uwsgi_master_fifo_prepare();
 int uwsgi_master_fifo();
 int uwsgi_master_fifo_manage(int);
 
-void uwsgi_log_do_rotate(char *, char *, off_t, int);
+void uwsgi_log_do_rotate(char *, char *, off_t, int, mode_t);
```

Comment:
Is this expected to be a public interface/maintain compatibility with any other existing code? If so I could create a new wrapper function and leave the original function signature alone.

Reply:
I'd say we should not worry https://github.com/unbit/uwsgi/commit/01a5611159825437235e7d98b8e0687937b523f8